pkg/beacon/relay/gjkr/protocol.go

```diff
@@ -906,27 +906,37 @@ func (sm *SharingMember) isValidMemberPublicKeySharePointsMessage(
 // What, using elliptic curve, is the same as:
 // G * s_ji == Σ ( A_j[k] * (i^k) ) for `k` in `[0..T]`
 func (sm *SharingMember) isShareValidAgainstPublicKeySharePoints(
-	senderID group.MemberIndex,
+	receiverID group.MemberIndex,
```

Comment:
I think I understand why we did this rename but would like to clarify. It is not about a *message receiver*, it is about a *share receiver*, that is, for which party `shareS` was generated for. Is that correct?
- In phase 8 (`VerifyPublicKeySharePoints`), we use this function passing message receiver (`sm.ID`)
- In phase 9 (`ResolvePublicKeySharePointsAccusationsMessages`), we use this function passing message sender (`message.senderID`)

If my understanding is correct, maybe we should rename `receiverID` to `shareReceiverID` and make it clear in the documentation. For example:

```
// isShareValidAgainstPublicKeySharePoints verifies if public key share points
// are valid for passed share S generated for the specific member, denoted as
// a share receiver.
```
?

Reply:
>  share receiver, that is, for which party shareS was generated for

We used to call them `peerID`, right?

Reply:
I am not sure, I see `peerID` is used only in `reconstructIndividualPrivateKeys`

Reply:
Yeah, agree with Piotr. `peerID` is a network concept used by libp2p and in terms of gjkr we should rather use `member`. Regarding `isShareValidAgainstPublicKeySharePoints` param I think `shareReceiverID` is the best proposition.